src/scripts/main.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, () => {
+      const message = `It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`;
```

Comment:
The message string includes the `element.nodeName` and `element.id`. Ensure that these properties are available and correctly set on the elements being passed to the `waitFor` function.